POEApi.Transport/HttpTransport.cs

```diff
@@ -206,11 +206,26 @@ public IWebProxy ProcessProxySettings()
         protected MemoryStream PerformHttpRequest(HttpMethod method, string url, bool? allowAutoRedirects = null,
             string requestData = null)
         {
-            using (var response = BuildHttpRequestAndGetResponse(method, url, allowAutoRedirects, requestData))
+            HttpWebResponse response = null;
+            // TODO: Don't retry an infinite number of times.
+            bool retry = true;
+            while (retry)

```

Comment:
Can we at least make this a controlled number of attempts, rather than an infinite loop.